src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 48px 46px 0;
+  // margin-bottom: 46px;
+
+  &__image {
+    padding: 32px 19px 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    padding: 40px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-text-black;
+  }
+
+  &__code {
+    margin: 4px 84px 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-text-grey;
+  }
+
+  &__price {
+    padding: 24px 16px 0;
+  }
+
+  &__link-buy {
+    margin: 16px;
+  }
+}
+
+.price-inside {
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+
+  &__text {
+    font-weight: 400;
+    font-size: 12px;
+    color: $color-text-grey;
+  }
+
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-text-black;
+  }
+}
+
+.link-buy {
+  box-sizing: border-box;
```

Comment:
move it to the * selector

Reply:
Hey! Don't leave unresolved comments, please.
If you have any questions, feel free to ask them in `fe_chat` in Slack.